src/components/map/layers/RoutePathLinkLayer.tsx

```diff
@@ -47,29 +47,32 @@ class RoutePathLayer extends Component<RoutePathLinkLayerProps> {
     }
 
     private renderNodes() {
+        const openPopup = (node: INode) => () => {
+            this.props.popupStore!.showPopup(node);
+        };
         const routePathLinks = this.props.routePathLinks;
-        return routePathLinks.map((routePathLink, index) => {
-            return (
-                <div key={index}>
-                    <NodeLayer
-                        key={`${routePathLink.startNode.id}`}
+        const nodes = routePathLinks
+            .map((routePathLink, index) => {
+                return (
+                    <NodeMarker
+                        key={`${routePathLink.startNode.id}-${index}`}
```

Comment:
What is -${index} supposed to do here? How is the startNode.id not unique enough?

Reply:
True, I guess it's not needed